src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -397,7 +397,12 @@ private function createBlock($messages, $type = null, $style = null, $prefix = '
                 $message = OutputFormatter::escape($message);
             }
 
-            $lines = array_merge($lines, explode(PHP_EOL, wordwrap($message, $this->lineLength - $prefixLength - $indentLength, PHP_EOL, true)));
+            $length = $this->lineLength;
+            if (false !== (bool) preg_match('(<\/.+>)', $message, $match)) {
```

Comment:
Why not use `!preg_match` here instead?

Reply:
It was to be strict, but we could change it to only preg_match instead.
done.